### ISSUE TITLE
Add "gained skills" to our community curator tiles

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#1903]
Additional Feature for https://fairsharing.org/community_curation/our_curators

At some point in the coming weeks, we are going to want to tag our curators with some additional skills/expertise tags from [terms4FAIRskills](https://obofoundry.org/ontology/t4fs)

These tags will be visible when the user clicks on the blue "+" icon in the top right of a curator's icon on the page. Each tag should be in the style of the "Early Adopter" tag that already exists, just visible from the blue overlay once the "+" is clicked. Below is a mockup of what I'm looking for please:

![image](https://user-images.githubusercontent.com/143586/199695080-a5138098-0510-4b04-96af-b54006a0e7da.png)

They will have the following fields (an example fictional tag is included):
Tag name|tag URL|date achieved
|---|---|---|
|using persistent identifiers|http://purl.obolibrary.org/obo/T4FS_0000492|20.10.2022

n.b. the "date achieved" will not be visible on the curator profile tiles, but we want to store it internally in case we want to refer back to it in future.

**Comment by knirirr** [comment]
What is the source of these data to be? Specifically, are we expected to get this directly from another site via an API or will the data be added to our users' accounts manually, e.g. by adding the fields to https://github.com/FAIRsharing/fairsharing.github.io/blob/master/src/data/communityCurationCohorts.json ?

If it's just the latter then it shouldn't be too difficult to include a field in each entry such as:

```
{
  ...
  "year_active": ["2022"],
  "gained_skills": [
    {
      "name": "using persistent identifiers",
      "date": "20.10.2022",
      "url": "http://purl.obolibrary.org/obo/T4FS_0000492"
    }
  ]   
},
```

Displaying it is another matter, of course...

**Comment by allysonlister** [comment]
I was only intending to add them directly to the json as described above, thanks!

**Comment by prakhyatox** [comment]
By when data for curators are expected?

**Comment by allysonlister** [comment]
As soon as the curators complete the requirements for gaining these tags - so it depends on how quickly they curate ;-)

**Comment by allysonlister** [comment]
Our first curator has met the challenge I set for getting the first tag. I realise that @prakhyatox is busy with other work, but perhaps someone else could take a look at this and complete it if possible?

Thanks!

**Comment by allysonlister** [comment]
Hi all,
 
The assistant has priority, but as part of the curation work associated with the assistant, we would like to offer curators a particular tag if they perform a certain level of curation. Therefore, once the assistant's first version is complete, please could this ticket be revisited so the addition of tags is possible?

thanks!

**Comment by ramGranell** [comment]
@allysonlister , we are planning to retake this ticket. I created some fictional data following format @knirirr 
 suggested: https://github.com/FAIRsharing/fairsharing.github.io/blob/gained_skills_%231903/src/data/communityCurationCohorts.json  However, would you mind providing real data for a couple of champions. Ideally, it would be: 
1 user id, name of the skill, date it was gained and url.
Are all the gained skills going to have an URL or is it possible that they don't have it?

**Comment by allysonlister** [comment]
Let's presume that all new skills will have associated URLs. Here's the second one I wanted to add:

http://purl.obolibrary.org/obo/TF4S_0000574 (metadata creation)

The other tag I wanted to add was for any curators who have really gone above and beyond in their champion role with us; this would be an additional tag in the same way as our existing "Early Adopter", and would likely NOT have an associated URL. This is just FYI as it wouldn't directly relate to this ticket.

Thanks!

**Comment by ramGranell** [comment]
Would you mind opening a new ticket about this? I think I can do this quite quick as it seems simple as I can do similar as "Early Adopter" The problem that I can have is there are two tags "Early adopter" and the new tag.

> The other tag I wanted to add was for any curators who have really gone above and beyond in their champion role with us; this would be an additional tag in the same way as our existing "Early Adopter", and would likely NOT have an associated URL. This is just FYI as it wouldn't directly relate to this ticket.

**Comment by ramGranell** [comment]
The code is created and mergeg. Skills to be created are as suggested in https://github.com/FAIRsharing/fairsharing.github.io/issues/1903#issuecomment-1315241269 e.g.
        "gained_skills": [
          {
            "name": "using persistent identifiers",
            "date": "20.10.2022",
            "url": "http://purl.obolibrary.org/obo/T4FS_0000492"
          },
          {
            "name": "Learning things",
            "date": "01.10.2022",
            "url": "http://purl.obolibrary.org/obo/T4FS_0000492"
          },
          {
            "name": "Being awesome",
            "date": "01.10.2022",
            "url": "http://purl.obolibrary.org/obo/T4FS_0000492"
          },
          {
            "name": "Volunteering for code testing",
            "date": "01.10.2022",
            "url": "http://purl.obolibrary.org/obo/T4FS_0000492"
          }
        ]
